apache-couchdb-mixin/rows.libsonnet

```diff
@@ -0,0 +1,93 @@
+          this.grafana.panels.overviewBulkRequestsPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.overviewResponseStatusOverviewPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.overviewGoodResponseStatusesPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.overviewErrorResponseStatusesPanel { gridPos+: { w: 12 } },
+        ],
+      ),
+
+    overviewReplication:
+      g.panel.row.new('Replication')
+      + g.panel.row.withCollapsed(false)
+      + g.panel.row.withPanels(
+        [
+          this.grafana.panels.overviewReplicatorChangesManagerDeathsPanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.overviewReplicatorChangesQueueDeathsPanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.overviewReplicatorChangesReaderDeathsPanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.overviewReplicatorConnectionOwnerCrashesPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.overviewReplicatorConnectionWorkerCrashesPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.overviewReplicatorJobsCrashesPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.overviewReplicatorJobsQueuedPanel { gridPos+: { w: 12 } },
+        ],
+      ),
+
+    nodes:
+      g.panel.row.new('Nodes')
+      + g.panel.row.withCollapsed(false)
+      + g.panel.row.withPanels(
+        [
+          this.grafana.panels.nodeErlangMemoryUsagePanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.nodeOpenOSFilesPanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.nodeOpenDatabasesPanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.nodeDatabaseWritesPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.nodeDatabaseReadsPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.nodeViewReadsPanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.nodeViewTimeoutsPanel { gridPos+: { h: 6, w: 8 } },
+          this.grafana.panels.nodeTemporaryViewReadsPanel { gridPos+: { h: 6, w: 8 } },
+        ],
+      ),
+
+    nodeRequests:
+      g.panel.row.new('Requests')
+      + g.panel.row.withCollapsed(false)
+      + g.panel.row.withPanels(
+        [
+          this.grafana.panels.nodeBulkRequestsPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.nodeAverageRequestLatencyPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.nodeRequestMethodsPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.nodeResponseStatusOverviewPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.nodeGoodResponseStatusesPanel { gridPos+: { w: 12 } },
+          this.grafana.panels.nodeErrorResponseStatusesPanel { gridPos+: { w: 12 } },
+        ],
+      ),
+
+    nodeLogs:
+      g.panel.row.new('Logs')
+      + g.panel.row.withCollapsed(false)
+      + g.panel.row.withPanels(
+        [
+          this.grafana.panels.nodeLogTypesPanel { gridPos+: { w: 24 } },
```

Comment:
I mentioned it separately, but could you please filter metrics with 0 values out for all of these panels in these two groups? 
<img width="964" height="659" alt="image" src="https://github.com/user-attachments/assets/f9974601-a9cd-436a-9cd7-7035d1aa2e4a" />
It's exceedingly busy, even with just one instance selected (ignore the label on the pie-chart, I was playing around there)